### PR TITLE
Allow admin to select organization from dropdown and fix dropdown

### LINKED ISSUE #520: Allow admin to select from dropdown for volunteer search and reports
## Description
- For now in admin account if you go for volunteer/search we see that we need to manually enter organization name which are limited and entering manually increases chances of mistake. 
- In administrator/report dropdown for job name and organization name is not displaying available job names and organization names.

### PR DESCRIPTION
**Pull request by pradeepgangwar**
- Fix dropdown in admin/reports where job name and organization name are not listed.
- Allow admin  to search a user with organization name selecting from dropdown.

Fixes #520

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@tapasweni-pathak Please review.

File: vms/volunteer/templates/volunteer/search.html [hunk 1]
```diff
@@ -137,7 +137,12 @@
                     <div class="form-group">
                         <label class="col-md-2 control-label">{% trans "Organization" %}</label>
                         <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Search by Organization{% endblocktrans %}" type="text" name="organization" value="{% if form.organization.value %}{{ form.organization.value }}{% endif %}">
+                            <select id="select" class="form-control" name="organization">
+								<option value="">-- {% trans "Select Organization" %} --</option>
```
Comment (tapasweni-pathak) [inline vms/volunteer/templates/volunteer/search.html]: 👋 Did you test this for any other language than English, say French?

Comment (pradeepgangwar) [reply vms/volunteer/templates/volunteer/search.html]: @tapasweni-pathak No. I didn't test for any other language. Please suggest how i proceed.

**Comment by pradeepgangwar** [comment]
@tapasweni-pathak  Please let me know it it requires more work.

File: vms/volunteer/templates/volunteer/search.html (hunk 1 above)
Comment (tapasweni-pathak) [reply vms/volunteer/templates/volunteer/search.html]: `trans` works here, just checked if we stick to `blocktrans` or not. Perfect!

**Pull request merged**